tensorbay/opendataset/BDD100K/loader.py

```diff
@@ -410,7 +406,9 @@ def _save_and_get_mask_info(
         if seg_type == "pan":
             all_category_ids = {}
         for instance_info in set(map(tuple, np.reshape(mask, (-1, 4)))):
-            instance_id = int(instance_info[-1])  # type:ignore[call-overload]
+            instance_id = (
+                int(instance_info[3]) + int(instance_info[2]) << 8  # type:ignore[call-overload]
```

Comment:
can we unpack the `instance_info` in line 408?
also the `int(instance_info[3]) + int(instance_info[2]) << 8` cannot be understood, may be add more comments here?